unittest/python/robot_wrapper.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+import os
+import unittest
+
+import pinocchio as pin
+
+
+class TestRobotWrapper(unittest.TestCase):
+    def setUp(self):
+        self.current_file = os.path.dirname(str(os.path.abspath(__file__)))
```

Comment:
This should use pathlib, but I see you got this from another test. I'll deal with it in another PR.

Reply:
OK thanks :smiley: